### PR TITLE
[enterprise-3.9] oc describe node shows less information

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by sjenning** [comment]
LGTM

**Comment by mburke5678** [comment]
@zhouying7780 PTAL

**Comment by zhouying7780** [comment]
LGTM

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL.

**Review by kalexand-rh — COMMENTED** [review]
LGTM

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -77,30 +77,77 @@ $ oc describe node <node>
...
- Container Runtime Version:	docker://1.6.0
- Kubelet Version:		v0.17.1-804-g496be63
- Kube-Proxy Version:		v0.17.1-804-g496be63
-ExternalID:			node1.example.com
-Pods:				(2 in total)
-  docker-registry-1-9yyw5
-  router-1-maytv
-No events.
-----
+# oc describe node vm252-51.gsslab.pnq2.redhat.com
```
Comment (vikram-redhat) [inline admin_guide/manage_nodes.adoc]: @mburke5678 - hey - can you please take out names of internal Red Hat instances and replace with generic hostnames throughout this file?

Comment (mburke5678) [reply admin_guide/manage_nodes.adoc]: @vikram-redhat PTAL to make sure I got all instances
3.4 https://github.com/openshift/openshift-docs/pull/10820
3.5 https://github.com/openshift/openshift-docs/pull/10821
3.6 https://github.com/openshift/openshift-docs/pull/10822
3.7 https://github.com/openshift/openshift-docs/pull/10823 
3.9 https://github.com/openshift/openshift-docs/pull/10824

Comment (vikram-redhat) [reply admin_guide/manage_nodes.adoc]: @mburke5678 - looks good, but why not do it in master and then CP? Unless this changes never made it to the master?

Comment (mburke5678) [reply admin_guide/manage_nodes.adoc]: @vikram-redhat Each version has different output for the command; some very different. I did each version from a separate branch in my repo and merged to the given version

Comment (vikram-redhat) [reply admin_guide/manage_nodes.adoc]: @mburke5678 - ah makes sense. What about master and 3.10 though?

Comment (mburke5678) [reply admin_guide/manage_nodes.adoc]: @vikram-redhat Good thoughts on master. Should the 3.9 output go to master? 

I will ask again about 3.10. I asked previously, but did not get a response.

Comment (vikram-redhat) [reply admin_guide/manage_nodes.adoc]: @mburke5678 master is the latest and greatest. You could run this on Origin using oc cluster up for Origin and test it that way to see what the output is.

Comment (mburke5678) [reply admin_guide/manage_nodes.adoc]: @vikram-redhat I merged to master via https://github.com/openshift/openshift-docs/pull/10837

**Pull request merged**